Userland/SampleCodeModule/loop.c

```diff
@@ -0,0 +1,11 @@
+#include <loop.h>
+#include <syscalls.h>
+#include <userstdlib.h>
+
+void loopProcess(void){
```

Comment:
If this is the entrypoint for a process, we should add the parameters to prevent warnings when using it
```suggestion
void loopProcess(int argc, char* argv[]){
```

Reply:
There are no warnings because I cast them when they are asigned to a process context info. Ill correct it anyway